chcemvediet/templates/main/homepage/casestudy/base.html

```diff
@@ -3,13 +3,18 @@
 {% comment %}
  %
  % Context:
- %  -- None
+ %  -- video: boolean
```

Comment:
Nepaci sa mi, ze informacia, ci sa ma video ikonka zobrazit alebo nie, je uvedena pri includovani itemu. Spravne by tato informacia mala byt uvedena vo vnutri itemu, tj. niekde vo vnutri `ako_pomaha_nas_portal.html` a podobne. Aby obsah itemu bol zapuzdreny vo vnutri toho suboru.

Nenapada mi, ako to spravit elegantne. Mozno jedno z rieseni by bolo celu ikonku definovat iba cez classu na `img`. Potom by stacilo spravit novy block:
```
<img class="chv-casestudy-img {% block img-class %}{% endblock %}" src="{% block img %}{% endblock %}">
```
A definicie itemov by mohli do bloku `img-class` pridat class ktora zobrazi ikonku.